client/reader/sidebar/index.jsx

```diff
@@ -49,6 +52,10 @@ const ReaderSidebar = React.createClass( {
 
 		// If we're browsing a tag or list, open the sidebar menu
 		this.openExpandableMenuForCurrentTagOrList();
+
+		if ( isEmpty( this.props.subscribedLists ) ) {
```

Comment:
We probably want to add a data component to handle the query. See `components/data/query-*` for some examples